kitebot/Jenkinsfile

```diff
@@ -215,6 +215,70 @@ pipeline {
                         }
                     }
                 }
+                stage('EL8.5 raw image') {
+                    when {
+                        beforeAgent true
+                        anyOf {
+                            anyOf {
+                                environment name: 'TRIGGERED_BY', value: 'Branch indexing'
+                                expression { return env.TRIGGERED_BY.matches("Pull request(.*)") }
+                            }
+                            allOf {
+                                environment name: 'TRIGGERED_BY', value: 'Triggered by CI message.'
+                                anyOf {
+                                    environment name: 'RELEASE_VERSION', value: '8.5.0'
+                                    environment name: 'RELEASE_VERSION', value: '8.5.1'
+                                }
+                            }
+                        }
+                    }
+                    agent { label 'vm-rhel-8-5-0' }
+                    environment {
+                        TEST_OS = "rhel-8-5"
+                    }
+                    steps {
+                        run_virt_test('ostree-raw')
```

Comment:
Could you please re-name it to `raw-image`. `ostree-raw` does not make any sense. Thanks.